src/Files.App/Views/Pages/PropertiesSecurity.xaml.cs

```diff
@@ -20,135 +20,135 @@
-			{
-				SecurityProperties = new SecurityProperties(listedItem);
-			}
-			else if (np.navParameter is DriveItem driveitem)
-			{
-				SecurityProperties = new SecurityProperties(driveitem);
-			}
-
-			base.OnNavigatedTo(e);
-		}
-
-		public async override Task<bool> SaveChangesAsync(ListedItem item)
-		{
-			if (SecurityProperties is not null)
-			{
-				return SecurityProperties.SetFilePermissions();
-			}
-			return true;
-		}
-
-		protected override void Properties_Loaded(object sender, RoutedEventArgs e)
-		{
-			base.Properties_Loaded(sender, e);
-
-			if (SecurityProperties is not null)
-			{
-				SecurityProperties.GetFilePermissions();
-			}
-		}
-
-		public override void Dispose()
-		{
-
-		}
-
-		private void OpenAdvancedProperties()
-		{
-			if (SecurityProperties is null)
-			{
-				return;
-			}
-
-			if (ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 8))
-			{
-				if (propsView is null)
-				{
-					var frame = new Frame();
-					frame.RequestedTheme = ThemeHelper.RootTheme;
-					frame.Navigate(typeof(PropertiesSecurityAdvanced), new PropertiesPageNavigationArguments()
-					{
-						Item = SecurityProperties.Item
-					}, new SuppressNavigationTransitionInfo());
-
-					// Initialize window
-					var propertiesWindow = new WinUIEx.WindowEx()
-					{
-						IsMinimizable = false,
-						IsMaximizable = false
-					};
-					var appWindow = propertiesWindow.AppWindow;
-
-					// Set icon
-					appWindow.SetIcon(FilePropertiesHelpers.GetFilesLogoPath());
-
-					// Set content
-					propertiesWindow.Content = frame;
-					if (frame.Content is PropertiesSecurityAdvanced properties)
-						properties.appWindow = appWindow;
-
-					// Set min size
-					propertiesWindow.MinWidth = 850;
-					propertiesWindow.MinHeight = 550;
-
-					// Set backdrop
-					propertiesWindow.Backdrop = new WinUIEx.MicaSystemBackdrop() { DarkTintOpacity = 0.8 };
-
-					appWindow.TitleBar.ExtendsContentIntoTitleBar = true;
-
-					// Set window buttons background to transparent
-					appWindow.TitleBar.ButtonBackgroundColor = Colors.Transparent;
-					appWindow.TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
-
-					appWindow.Title = string.Format("SecurityAdvancedPermissionsTitle".GetLocalizedResource(), SecurityProperties.Item.Name);
-					appWindow.Resize(new SizeInt32(850, 550));
-					appWindow.Destroying += AppWindow_Destroying;
-					appWindow.Show();
-
-					propsView = appWindow;
-				}
-				else
-				{
-					propsView.Show(true);
-				}
-			}
-			else
-			{
-				// Unsupported
-			}
-		}
-
-		private async void AppWindow_Destroying(AppWindow sender, object args)
-		{
-			sender.Destroying -= AppWindow_Destroying;
-			propsView = null;
-
-			if (SecurityProperties is not null)
-			{
-				await DispatcherQueue.EnqueueAsync(() => SecurityProperties.GetFilePermissions()); // Reload permissions
-			}
-		}
-	}
+    public sealed partial class PropertiesSecurity : PropertiesTab
```

Comment:
The whole indentation of the file has been move from tabs to spaces. It should be put back to tabs.

Reply:
It's done.